src/test/run-pass/borrowck/two-phase-bin-ops.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// revisions: lxl nll
+//[lxl]compile-flags: -Z borrowck=mir -Z two-phase-borrows
```

Comment:
Actually, this `lxl` revision is still using the MIR borrowck. Can we add a revision that tests the existing borrowck (possibly replacing the existing `lxl`)? e.g.

```
// Test that NLL and the older, AST-based checker both accept these constructions:
// revisions: ast nll

#![cfg_attr(nll, feature(nll))]
```

Reply:
(I don't know that we care so much about the MIR-based borrowck without NLL enabled)

Reply:
Makes sense, but now I'm really confused about the arguments specified here https://github.com/rust-lang/rust/blob/3ec5a99aaa0084d97a9e845b34fdf03d1462c475/src/test/run-pass/borrowck/two-phase-control-flow-split-before-activation.rs#L12

In any case, I've dropped those args now.

Reply:
Well, I think that felix was specifically targeting MIR borrowck w/o NLL. But in this case, I don't think we need to, but I think it's important to show that we are preserving the current behavior in terms of what code is accepted.